### DISCUSSION TITLE
Most underrated framework ever, why not enough samples & demos.

### DISCUSSION THREAD

**Discussion by theHamdiz** [RibirX/Ribir#600] [Ideas]
Hello, is it just me or this is really the most underrated rust-based gui framework ever?

Also why there's not enough toy applications & various components shown in the official documentation or here on GitHub?

**Comment by M-Adoo** [comment]
We continue to concentrate on the core features and API design of Ribir. While most of the work is complete, we aim to refine these features to make them more user-friendly and stable. We're excited to add more examples and widgets in the next phase, which may start in July or August. Of course, we warmly welcome more people to join us in improving Ribir.
